Created pull request template (#225)
emmo-repo/EMMO-python: .github/pull_request_template.md

# Description:

> _Summary of change, including the issue(s) to be adressed._

Reviewer: ```suggestion
<!-- Summary of change, including the issue(s) to be addressed. -->
```

## Type of change:

> _Put an `x` in the box that applies._

Reviewer: ```suggestion
<!-- Put an `x` in the box that applies. -->
```

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist:

> _Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

Reviewer: ```suggestion
<!-- Put an `x` in the boxes that apply. You can also fill these out after creating the PR. -->
```

> This checklist can be used as a help for the reviewer.

Reviewer: ```suggestion
<!-- This checklist can be used as a help for the reviewer. -->
```
Maybe you want this shown in the PR though?

Author: Yes, this should stay visible I think.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?

> - [ ] Does a new feature has an accompanying new test in the CI/CD?

Reviewer: ```suggestion
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
```

- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?

> - [ ] Does the new feature provide new restrictions on dependecies, and if so is this documented?

Reviewer: ```suggestion
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?
```

## Comments:

> _Additional comments here, including clarifications on checklist if applicable._

Reviewer: ```suggestion
<!-- Additional comments here, including clarifications on checklist if applicable. -->
```
